Tests/CurrencyTests/CurrencyValue+StringRepresentationTests.swift

```diff
@@ -149,15 +150,15 @@ extension CurrencyValueStringRepresentationTests {
   }
 
   func test_stringInterpolation_forLocale_usesDefaultLocale() {
-    XCTAssertEqual("\(localize: USD(4321.389))", "$4,321.39")
+    XCTAssertEqual("\(localize: USD(4321.389), for: Locale(identifier: "en_US"))", "$4,321.39")
   }
 }
 
 // MARK: LocalizedString
 
 extension CurrencyValueStringRepresentationTests {
   func test_localizedString_forLocale_usesDefaultLocale() {
-    XCTAssertEqual(USD(4321.389).localizedString(), "$4,321.39")
+    XCTAssertEqual(USD(4321.389).localizedString(for: .init(identifier: "en_US")), "$4,321.39")
```

Comment:
🤔 I'm honestly thinking we just delete this test + `test_stringInterpolation_forLocale_usesDefaultLocale` because we're effectively testing that the Swift language properly uses default values.

There IS value of having a test that asserts WHAT the default value is, but I think we need to rewrite the tests to basically duplicate the formatting logic with the `Locale.current` as the hard coded value, and then comparing the output

eg

```swift
let formatter = ...
formatter.locale = .current
let expectedOutput = formatter.string(for: ...)
XCTAssertEqual(...localizedString(), expectedOutput)
```

Reply:
Because as-is, this change of passing in a locale explicitly just duplicates other unit tests we already have